index.js

```diff
@@ -9,7 +9,7 @@ var ExtractedModule = require("./ExtractedModule");
 var Chunk = require("webpack/lib/Chunk");
 var OrderUndefinedError = require("./OrderUndefinedError");
 var loaderUtils = require("loader-utils");
-var schemaTester = require('./schema/validator');
+var validateOptions = require('schema-utils');
 var loaderSchema = require('./schema/loader-schema');
```

Comment:
The two schemas can be removed, just pass the `path/to/schema.json` to `validateOptions`